src/index.ts

```diff
@@ -5,42 +5,40 @@
  * LICENSE file in the root directory of this source tree.
  *
  */
+import { NativeModules, Platform } from 'react-native';
+import type { Spec } from './NativeRNCImageEditor';
 
-import { NativeModules } from 'react-native';
+const LINKING_ERROR =
+  `The package '@react-native-community/image-editor' doesn't seem to be linked. Make sure: \n\n` +
+  Platform.select({ ios: "- You have run 'pod install'\n", default: '' }) +
+  '- You rebuilt the app after installing the package\n' +
+  '- You are not using Expo Go\n';
 
-const { RNCImageEditor } = NativeModules;
+// @ts-expect-error
+const isTurboModuleEnabled = global.__turboModuleProxy != null;
 
-type $Maybe<T> = T | null | undefined;
+const RNCImageEditorModule = isTurboModuleEnabled
+  ? require('./NativeRNCImageEditor').default
+  : NativeModules.RNCImageEditor;
```

Comment:
All those added lines are not needed, because `TurboModuleRegistry.getEnforcing` under the hood already falls back to `NativeModules.RNCImageEditor`

Reply:
this is an output of `create-react-native-library@latest` template
that I used for adding new arch support

(you can open an issue if it a bug https://github.com/callstack/react-native-builder-bob/issues)

Reply:
I'll file a PR next week, but in the meantime, please simplify that chunk of code, so that it always imports the TurboModuleRegistry exported module

Reply:
done

Reply:
done